graph-manager-docs/source/client-awareness.md

```diff
@@ -1,23 +1,26 @@
+Apollo Graph Manager enables you to view operation metrics for each of your application's different clients (such as `web` and `iOS`) and client versions (such as `1.0` and `1.1`), helping you understand how each one interacts with your data graph. This feature (called **client awareness**) is especially useful as your number of clients grows, and as you consider when to deprecate or discontinue support for older client versions. [See other common use cases](#common-use-cases)
 
-Apollo provides a client identification and tracking system, which allows you to answer questions like _"which query is using this field?"_ and _"which versions of my iOS app are running this query?"_. It segments usage data by **client name and version** and allows for **field-level understanding** of how consumers are interacting with your graph in real-time.
+![client overview](./img/client-awareness/overview.png)
 
-Like any API, your graph will end up with many consumers with different frequencies, subselections, and permissions as it grows over time. Apollo allows all reported data to be tagged with client information so it can be filtered and analyzed across different sets of clients and stacks.
+## Setup
 
-Here's an example of client identity reporting in Apollo Graph Manager:
+First, make sure that your GraphQL server is already [pushing analytics to Graph Manager](./setup-analytics).
 
-![client overview](./img/client-awareness/overview.png)
+### Using Apollo Server and Apollo Client
 
-## Setup
+By default, Apollo Server checks for the presence of the following HTTP headers in every incoming operation request:
+
+* `apollographql-client-name`
+* `apollographql-client-version`
 
-Apollo Server 2.2.3+ will look for specific request headers, `apollographql-client-name` and `apollographql-client-version`, by default. If present, Apollo Server will extract them and make sure the data for that request is reported to Apollo's systems with the correct client and version tag.
+If one or both of these headers are present, Apollo Server automatically extracts their values and includes them in the trace report that it sends to Graph Manager. You can [override this default behavior](#advanced-server-configuration).
 
-With Apollo Client 2.4.6+, simply passing the `name` and `version` options in your `ApolloClient` constructor will automatically add these headers to every request. Setting up client identity reporting is as simple as adding configuration to Apollo Client:
+If you're using Apollo Client, you can provide the `name` and `version` options to the `ApolloClient` constructor. If you do, Apollo Client automatically assigns the provided values to the corresponding headers in every operation request.
```

Comment:
It would be nice if we could have the information about ApolloClient usage earlier so that users don't need to read all of the implementation details. It would also be nice if there was built-in support for Apollo iOS and others (and there might be!)

Maybe in lieu of that, we could **bold** Apollo Client to call attention to it for AC users?